[DOCS] MITRE ATT&CK being added to alert details flyout and alert reason re-named (#2772)
elastic/security-docs: docs/detections/alerts-view-details.asciidoc

* *Summary*: Displays general details such as the alert's status, severity, risk score, and a link to the detection rule that produced the alert.

> * *MITRE ATT&CK*: Provides relevant https://attack.mitre.org/[MITRE] framework tactics, techniques, and subtechniques.

Reviewer: Nit: I think the full name would be best here. "MITRE" is the name of the organization that does the research, and "ATT&CK" is the name of their knowledge-base.
```suggestion
* *MITRE ATT&CK*: Provides relevant https://attack.mitre.org/[MITRE ATT&CK] framework tactics, techniques, and subtechniques.
```

Author: Makes sense, I'll add this change - thanks!

> * *Alert reason*: Provides alert details in a logical arrangement. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.

Author: ```suggestion
* *Alert reason*: Provides alert details in a logical arrangement. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive; hover over one to access the available actions.
```

> * *Alert reason*: Provides alert details in a logically-arranged format. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.

Reviewer 2: ```suggestion
* *Alert reason*: Provides alert details in a logical arrangement. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.
```
